cmake/intel_flags.cmake

```diff
@@ -0,0 +1,33 @@
+
+message(STATUS "Getting ifort flags")
+
+if(WIN32)
+  set(CMAKE_Fortran_FLAGS_RELEASE "${CMAKE_Fortran_FLAGS_RELEASE} -nologo -fpp -O3 -heap-arrays1024 -QaxCORE-AVX2,CORE-AVX-I,AVX,SSE4.2,SSSE3 -Qipo -fp:fast=2 -Qdiag-disable:remark -Qmkl")
```

Comment:
Normally CMake handles these flags fine specifically for Intel compiler, including on Windows. What about older CPUs?  I typically use for Intel compiler `-march=native` or `/arch:native` instead of all these flags.

Note: For Gfortran, `-march=native` does not work in general, `-mtune=native` should be used instead.